docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -1825,6 +1825,68 @@ end
 
 * https://api.rubyonrails.org/classes/AbstractController/Callbacks/ClassMethods.html#method-i-_normalize_callback_options
 
+== Rails/ImplementedMigration
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Disabled
+| Yes
+| No
+| 2.10
+| -
+|===
+
+This cop checks whether the migration implements
+either a `change` method or both an `up` and a `down`
+method.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+class SomeMigration < ActiveRecord::Migration[6.0]
+  def chance # <----- typo
+    # migration
+  end
+end
```

Comment:
I think typos are not the main focus of the cop, so con you remove this example?